docs/tutorials/creating-your-first-substrate-chain/setup.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Setup Your Computer
+---
+
+Normally we would teach you more about the Substrate blockchain development
+framework and the Proof of Existence blockchain you will be building. However,
+setting up your computer for Substrate development can take a while. 
+
+To optimize your time, we will have you start the setup process. In the next
+section, while things are compiling, we will then tell you more about Substrate
+and what we are building.
+
+## Prerequisites
+
+To develop on Substrate, your computer needs some prerequisites to establish a
+working development environment.
+
+> **Note:** Setting up your computer is probably the hardest part of this
+> tutorial, so don't let this discourage you.
+
+### Substrate Development
+
+If you are using a Unix based machine (Linux, MacOS), we have created a simple
+one-liner to get all of those prerequisites installed for you:
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+> **Note:** If you want to see specifically what this script does just visit:
+> https://getsubstrate.io
+
+It will automatically install:
+
+* [CMake](https://cmake.org/install/)
+* [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/)
+* [OpenSSL](https://www.openssl.org/)
+* [Git](https://git-scm.com/downloads)
+* [Rust](https://www.rust-lang.org/tools/install)
+
+If you are using Windows and do not have the [Windows Subsystem for
+Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10), the process
+is a bit harder, but well documented [here](getting-started.md).
+
+### Front-End Development
+
+This tutorial also uses a custom ReactJS front-end which we will modify for
+interacting with our custom Substrate blockchain.
+
+To use the front-end project, you need to install [Yarn](https://yarnpkg.com),
+which may also require you separately install [Node.js](https://nodejs.org/).
+
+* [Install Node.js](https://nodejs.org/en/download/)
+* [Install Yarn](https://yarnpkg.com/lang/en/docs/install/)
+
+## Compiling Substrate
+
+We have created simple template projects to help you get started building on
+Substrate.
+
+1. Clone the Substrate Node Template and Substrate Front-End Template
+
+    ```bash
+    git clone https://github.com/substrate-developer-hub/substrate-node-template
+    git clone https://github.com/substrate-developer-hub/substrate-front-end-template
```

Comment:
We should make sure these they get compatible versions by either using package, or having them check out specific tags in the constituent projects. I recently cut version 1.5.0 in package and all constituents, and revised the release plan.

Reply:
Ideally I would rather not make these lines more complicated than they have to. I would be willing to front any burden of making sure that the guide always works with the master version of this git repository (which I think will be the case anyway with 2.0 release)

Reply:
I could be convinced off this position, but I really love that I can say to a person how to start a new substrate project with east. "You git clone from github, substrate developer hub, substrate node template"